src/platform/forms/save-in-progress/RoutedSavableApp.jsx

```diff
@@ -155,10 +155,10 @@ class RoutedSavableApp extends React.Component {
 
   onbeforeunload = e => {
     const { currentLocation, autoSavedStatus, formConfig } = this.props;
-    const { additionalRoutes = [] } = formConfig;
+    const additionalRoutes = formConfig?.additionalRoutes;
```

Comment:
To be clear, I'm not against all instances of destructuring. I think in this case, the original was better. Also, it had a default value of `[]`.

Reply:
> Is there a reason to go with `??` instead of `||`?
> 
> I asked the question earlier if it made sense to allow for falsy values like an empty string, but didn't get any answers. My impression is that we would rather go with `||` because I'd expect the bug that this was supposed to fix could still occur if we went with `??` (displays empty string instead of the default string). But I don't work on forms regularly, so I was just looking for confirmation whether we should allow something like that.

This was taken from @erikphansen 's suggestion [here](https://github.com/department-of-veterans-affairs/vets-website/pull/13689#discussion_r467096269).  But if you feel that || is safer, I can implement that instead.

Reply:
I would think `||` makes more sense.